Dockerfile

```diff
@@ -282,13 +283,19 @@ VOLUME /tmp
 ENV CGO_ENABLED=0
 # copy cosign binary for integration test
 COPY --from=gcr.io/projectsigstore/cosign:v2.0.0@sha256:728944a9542a7235b4358c4ab2bcea855840e9d4b9594febca5c2207f5da7f38 /ko-app/cosign /usr/local/bin/cosign
+# installing soci for integration test
```

Comment:
We can create a new toml file like `test_integration_etc_containerd_config,toml` including both stargz and soci config, then copy it to overwrite the original etc_containerd_config.toml in test-integration stage.
If you have any script to overwrite, you can also use it.

Then do `systemctl restart containerd` after it.